### PR TITLE
Nature demo blog post

### PR DESCRIPTION
**Pull request by rgbkrk**
Now that I've broken this up, I'm making this one solely about Nature. This is a Work In Progress, so no shipping just yet.

I still have some more bits to fill in that I'll address when I get home tonight. I'm guessing we'll need to change the date to tomorrow and ship it then.

### CODE REVIEW

File: src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+![Redirect tmpnb nodes](https://cloud.githubusercontent.com/assets/836375/5913707/3b282e20-a5ae-11e4-9768-c22089b70497.png)
+
+## Closing up
+
+In the end we ended up serving more than 20,000 notebook servers and counting. 
+
+We love IPython notebooks, the overall architecture that has been built out here, and hope to keep supporting Open Source projects do interesting things on the internet in a way that benefits community, technology, and the whole ecosystem.
+
+We didn't expect to build a system that would let people create user environments they could use for teaching and tutorials.
```
Comment (smashwilson) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: I would reword or remove this very last sentence. As it is, it feels a little out of left field based on the rest of the article: it's the only time you allude to the continued use of tmpnb post-Nature.

If you do want to keep it (and it is a nice "into the future" bit), I'd recommend fleshing it out just a bit. "Since the Nature article went live, others have found more uses for tmpnb! For example..." And then conclude with this sentence, maybe with an addition like "... but we're delighted to see it be useful in unexpected ways."

**Comment by smashwilson** [comment]
:+1: Sounds good to me! Other than the conclusion I think it's ready to go.

File: src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md (hunk 1 above)
Comment (rgbkrk) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Oh gosh, didn't mean to leave that anymore. I just deleted it.

File: src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "How did we serve > 20,000 IPython notebooks for Nature readers?"
```
Comment (rdodev) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Not super crazy about the title. Nothing wrong with it, tbh. Just needs more pizzazz. I'll come up with suggestions for consideration in a bit.

Comment (rdodev) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Some suggestions (again just for consideration):
 - "Delivering IPython Notebooks At Web Scale"
 - "I spun 20,000 notebooks and I'd spin 20,000 more" ---- ok that was pretty bad
 - "A Thousand Score IPython Notebooks Served For Nature Readers"

OK I'm out of bad ideas. :disappointed:

Comment (rgbkrk) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Would it better if I wrote "more" instead of ">"?

Comment (rdodev) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Yes.

Comment (rgbkrk) [inline src/site_source/_posts/2015-01-27-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Done. Updated date too.

File: src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    - Science
+    - Data Science
+    - Docker
+    - tmpnb
+    - Jupyter
+    - Ephemeral
+---
+
+The IPython/Jupyter notebook is a wonderful environment for computations, prose, plots, and interactive widgets that you can share with collaborators. People use the notebook [all](http://blog.quantopian.com/quantopian-research-your-backtesting-data-meets-ipython-notebook/) [over](http://nbviewer.ipython.org/github/GoogleCloudPlatform/ipython-soccer-predictions/blob/master/predict/wc-final.ipynb) [the](https://github.com/facebook/iTorch) [place](http://nbviewer.ipython.org/url/norvig.com/ipython/TSPv3.ipynb) across [many varied languages](https://github.com/ipython/ipython/wiki/IPython-kernels-for-other-languages). It gets used by [data scientists](http://nbviewer.ipython.org/gist/wrobstory/1eb8cb704a52d18b9ee8/Up%20and%20Down%20PyData%202014.ipynb), researchers, analysts, developers, and people in between.
+
```
Comment (rdodev) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: please add `<!-- more -->` here. One blank line above, one below.

Comment (rgbkrk) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: I can delete the empty lines too. Do you prefer the HTML comment instead?

Comment (smashwilson) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: Jekyll uses the HTML comment to determine what to use as an excerpt on the front page (hence the pickiness about surrounding whitespace). :stars:

Comment (rgbkrk) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: ohhhhhh, right

Comment (rgbkrk) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: This makes me realize the intro is more about introducing IPython more than it is about introducing the article. Should I have a different summary piece here?

Comment (rdodev) [inline src/site_source/_posts/2015-01-28-how-did-we-serve-more-than-20000-ipython-notebooks-for-nature.md]: I think the intro to IPython notebook is relevant and appropriate.

My 2c

**Comment by rdodev** [comment]
OK! This looks ready to roll! :shipit:
